backend/src/main/java/com/bakdata/conquery/resources/admin/ui/ConceptsUIResource.java

```diff
@@ -32,4 +36,14 @@ public View getConceptView() {
 			concept
 		);
 	}
+
+	@DELETE
+	public void removeConcept() throws IOException, JSONException {
+		namespace.getStorage().removeConcept(conceptId);
```

Comment:
In die UI Klassen sollten eigentlich nur Endpunkte die ein HTML zurückgeben. Dieser Endpunkt sollte in das `api` Subpackage (`package com.bakdata.conquery.resources.api;`) in die Klasse `ConceptResource`.

Die Löschfunktionalität selbst sollte in die `ConceptsProcessor` Klasse.